### ISSUE TITLE
Use of uninitialized value $config/$policy

### ISSUE THREAD

**Issue by matsduf** [dotse/zonemaster-backend#268]
The Zonemaster Backend generates log messages due to errors in code:

```
Use of uninitialized value $config in pattern match (m//) at /usr/local/share/perl/5.18.2/Zonemaster/WebBackend/Engine.pm line 356.
Use of uninitialized value $config in regexp compilation at /usr/local/share/perl/5.18.2/Zonemaster/WebBackend/Engine.pm line 357.
Use of uninitialized value $policy in pattern match (m//) at /usr/local/share/perl/5.18.2/Zonemaster/WebBackend/Engine.pm line 350.
Use of uninitialized value $policy in regexp compilation at /usr/local/share/perl/5.18.2/Zonemaster/WebBackend/Engine.pm line 351.

```

**Comment by sandoche2k** [comment]
Refer to https://github.com/dotse/zonemaster/issues/498

**Comment by sandoche2k** [comment]
closed https://github.com/dotse/zonemaster-backend/pull/329

**Comment by matsduf** [comment]
Fixed by #329
